### DISCUSSION TITLE
Does anyone can "brew bump-formula-pr vte3" ?

### DISCUSSION THREAD

**Discussion by inoueta** [Homebrew/discussions#598] [Everyday usage]
New source of vte3 is now available at the same place as the previous release.
https://download.gnome.org/sources/vte/0.62/

So, I want to request to catch up it. But I do not have GitHub token set up.
I'm very appreciated if someone can do it for us.  Thank you.

**Comment by carlocab** [comment]
Updated in https://github.com/Homebrew/homebrew-core/commit/d665cfdc3151beecc3a89e90948ecf2994b7dfd5.

**Reply by inoueta** [reply]
Thanks!
